packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -223,21 +219,24 @@ export const prepareAdvancerKit = (
             });
           } catch (error) {
             log('Advancer error:', error);
-            statusManager.skipAdvance(evidence, [error.message]);
+            statusManager.skipAdvance(evidence, [(error as Error).message]);
           }
         },
         /** @param {CosmosChainAddress} intermediateRecipient */
-        setIntermediateRecipient(intermediateRecipient) {
+        setIntermediateRecipient(intermediateRecipient: CosmosChainAddress) {
           this.state.intermediateRecipient = intermediateRecipient;
         },
       },
       depositHandler: {
         /**
-         * @param {undefined} result
-         * @param {AdvancerVowCtx & { tmpSeat: ZCFSeat }} ctx
+         * @param result
+         * @param ctx
```

Comment:
do we have an `XXX` or an issue or something about how our lint config requires these useless `@param` decls?

Reply:
when it came up last the decision was to keep them as a starting point for documenting.

I'm okay with that so far but if you want it to go away please file an issue or start a thread